src/Persistence/ConfigDeserializer.php

```diff
@@ -35,10 +35,16 @@ public function deserialize( string $configJson ): Config {
 	 * @param array<string, mixed> $configArray
 	 */
 	private function newConfig( array $configArray ): Config {
+		// Need to stop phpstan from complaining about the type of $configPerItemType
+		$configPerItemType = isset( $configArray['configPerItemType'] ) && is_array( $configArray['configPerItemType'] )
+			? $configArray['configPerItemType']
+			: [];
+
```

Comment:
I rather not add such code that is there only to make a tool happy, at least in cases where we can easily fix the issue on the tool side by changing config or adding the issue to the baseline file.

I like you added the comment

Reply:
You can update the baseline file with `make stan-baseline` from within the container